presets/okta/syslog/preset.yaml

```diff
@@ -0,0 +1,695 @@
+                WHEN LOWER(eventType) = 'user.authentication.sso' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.authentication.verify' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.lifecycle.deactivate' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'user.lifecycle.suspend' THEN 'Account Change'
+                WHEN LOWER(eventType) = 'user.mfa.factor.activate' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.mfa.factor.update' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.session.access_admin_app' THEN 'Authentication'
+                WHEN LOWER(eventType) = 'user.session.clear' THEN 'Authorize Session'
+                WHEN LOWER(eventType) = 'user.session.start' THEN 'Authorize Session'
+                ELSE NULL
+            END
+        - name: ocsf_activity_id
+          expr: |
+            CASE
+                WHEN LOWER(eventType) = 'app.user_management.grouppush.mapping.okta.users.ignored' THEN CAST('3' AS INT)
+                ELSE CAST('99' AS INT)
+            END
+        - name: ocsf_activity_name
+          expr: |
+            CASE
+                WHEN LOWER(eventType) = 'app.user_management.grouppush.mapping.okta.users.ignored' THEN 'update'
+                ELSE 'other'
+            END
+        - name: ocsf_actor.user.uid
+          expr: actor.id
+        - name: ocsf_actor.user.name
+          expr: actor.displayName
+        - name: ocsf_actor.user.email_addr
+          expr: actor.alternateId
+        - name: ocsf_actor.user.uid_alt
+          expr: actor.alternateId
+        - name: ocsf_actor.session.uid
+          expr: authenticationContext.externalSessionId
+        - name: ocsf_actor.session.issuer
+          expr: authenticationContext.issuer.id
+        - name: ocsf_actor.user.type
+          expr: |
+            CASE 
+                WHEN LOWER(actor.type) = 'user' THEN 'user'
+                WHEN LOWER(actor.type) = 'publicclientapp' THEN 'other'
+                WHEN LOWER(actor.type) = 'systemprincipal' THEN 'system'
+                ELSE 'other'
+            END
+        - name: ocsf_actor.user.type_id
+          expr: |
+            CASE 
+                WHEN LOWER(actor.type) = 'user' THEN CAST('1' AS INT)
+                WHEN LOWER(actor.type) = 'publicclientapp' THEN CAST('99' AS INT)
+                WHEN LOWER(actor.type) = 'systemprincipal' THEN CAST('3' AS INT)
+                ELSE CAST('99' AS INT)
+            END
+        - name: ocsf_status_id
+          expr: |
+            CASE 
+                WHEN LOWER(outcome.result) = 'allow' THEN CAST('1' AS INT)
+                WHEN LOWER(outcome.result) = 'success' THEN CAST('1' AS INT)
+                WHEN LOWER(outcome.result) = 'deny' THEN CAST('2' AS INT)
+                WHEN LOWER(outcome.result) = 'failure' THEN CAST('2' AS INT)
+                ELSE CAST('99' AS INT)
+            END
```

Comment:
`status_id` is a partnered attribute so we should be adding `status` here as well.